### PR TITLE
LG-2505: Refresh radio box group

### PR DESCRIPTION
**Pull request by shaneeza**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-2505](https://jira.mongodb.org/browse/LG-2505)

Screen Shot 2022-10-18 at 2 50 40 PM
Screen Shot 2022-10-18 at 2 50 48 PM

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Style change (purely visual updates; if this is the case, attach a screenshot below!)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

## Feedback Levels

Please try to mark your comments with one of these levels

1. Praise (👏 🚀 🔥 🍾)
2. Opinions, Ideas, Suggestions (💡 👀)
3. Let’s discuss @ Code review (🤷 🤔 🧐 )
4. Doesn’t meet agreed upon standards (😬 ⚠️ ‼️)
5. Missing an edge case (🛑 🙅 ❌)
6. Missing core functionality (⛔️ ☣️ 🚧 🌋 💀)

(Note: Use ⌘-⌃-␣ to open the Emoji keyboard!)

### CODE REVIEW

File: packages/radio-box-group/src/RadioBox.styles.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  darkMode: boolean;
+}
+
+export const getRadioDisplayStyles = ({
+  checked,
+  disabled,
+  size,
+  showFocus,
+  darkMode,
+}: StateForStyles) => {
```
Comment (TheSonOfThomp) [inline packages/radio-box-group/src/RadioBox.styles.ts:41]: Not a huge fan of this pattern, but it works 👍

**Review by TheSonOfThomp — APPROVED** [review]
LGTM 🚀

File: packages/radio-box-group/src/RadioBox.styles.ts (hunk 1 above)
Comment (shaneeza) [reply packages/radio-box-group/src/RadioBox.styles.ts:41]: I mostly copied these styles over to a new style file

File: packages/radio-box-group/src/RadioBoxGroup.spec.tsx [hunk 2]
```diff
@@ -92,9 +92,13 @@ describe('packages/RadioBoxGroup', () => {
...
-  test.todo(`correct number of children render`);
-  // expect(radioBoxGroupContainer.children).toHaveLength(4)
-  // radioBoxGroupContainer.children.length returns 1
+  const allChildren = radioBoxGroupContainer.children.length;
+
+  test(`correct number of children render`, () => {
+    // expect(radioBoxGroupContainer.children).toHaveLength(4)
+    // inside this test radioBoxGroupContainer.children.length returns 3 but outside it returns 4
+    expect(allChildren).toEqual(4);
+  });
```
[the remark this answers is not in the document]
Comment (shaneeza) [reply packages/radio-box-group/src/RadioBoxGroup.spec.tsx:101]: yeah i don't understand why, the output text shows 4 items in the array but the returns 3

**Pull request merged**